jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -616,6 +618,51 @@ public boolean removeEventListener(EventListener listener)
         return false;
     }
 
+    @Override
+    public void initializeDefaults(Attributes attributes)
+    {
+        for (String keyName : attributes.getAttributeNameSet())
+        {
+            Object value = attributes.getAttribute(keyName);
+            if (LOG.isDebugEnabled())
+                LOG.debug("init {}: {}", keyName, value);
+
+            switch (keyName)
+            {
+                case Deployable.TEMP_DIR -> setTempDirectory(IO.asFile(value));
```

Comment:
maybe go via a Resource so it can be specified either in local FS file syntax or URI file: syntax

Reply:
Done, but in IO.asFile() instead.